chutney/ui/src/app/modules/campaign/components/execution/detail/campaign-execution.component.scss

```diff
@@ -0,0 +1,5 @@
+:host ::ng-deep .popover-body {
```

Comment:
use [custom class](https://ng-bootstrap.github.io/#/components/popover/examples#customclass) instead

Reply:
custom class doesn't work when it's used inside nested angular components.
The only solution was to define the custom class in syle.scss file.